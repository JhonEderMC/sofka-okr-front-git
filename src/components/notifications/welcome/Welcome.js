import React, { useState } from 'react';
import Modal from 'react-modal';
import '../notificationsWindows.scss'

const Welcome = () => {
    const [modalIsOpen, setModalIsOpen] = useState(false)

    let nombre = "Andres Caro"

    const customStyles = {
        content: {
            top: '20%',
            left: '50%',
            right: 'auto',
            bottom: 'auto',
            marginRight: '-50%',
            width: '600px',
            transform: 'translate(-40%, -10%)',
            padding: '0',
        },
    };

    return (
        <div id="root">
            <button onClick={() => setModalIsOpen(true)}>Crear un Usuario</button>
            <Modal isOpen={modalIsOpen}
                onRequestClose={() => setModalIsOpen(false)}
                style={customStyles}>
                <header className='headerNotificationWindows' id='headerNotificationWindows'></header>
                <div className='bodyNotificationWindows' id='bodyNotificationWindows'>
                    <h1>Â¡Bienvenido {nombre}! </h1>
                    <p>"Estamos felices de tenerte como un nuevo integrante, esperamos que podamos aprender de tus aportes"</p>
                </div>
                <div className='imageNotificationWindows' id='imageNotificationWindows'>
                    <img src="https://www.sofka.com.co/wp-content/uploads/2020/08/sofka-logo-gradient-white.png" alt="logo-sofka"></img>
                </div>
                <footer className='footerNotificationWindows' id='footerNotificationWindows'></footer>
            </Modal>
        </div>
    )
}

export default Welcome;