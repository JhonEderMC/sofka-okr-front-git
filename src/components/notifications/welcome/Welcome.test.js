import React from 'react';
import Modal from 'react-modal';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import Welcome from './Welcome'
import sinon from 'sinon'

configure({ adapter: new Adapter() });

it('show content of modal', ()=>{
})

