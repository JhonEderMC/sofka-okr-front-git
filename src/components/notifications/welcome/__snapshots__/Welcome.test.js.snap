// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders childen when modal is open 1`] = `
Array [
  <header
    className="headerNotificationWindows"
    id="headerNotificationWindows"
  />,
  <div
    className="bodyNotificationWindows"
    id="bodyNotificationWindows"
  >
    <h1>
      Â¡Bienvenido 
      Andres Caro
      ! 
    </h1>
    <p>
      "Estamos felices de tenerte como un nuevo integrante, esperamos que podamos aprender de tus aportes"
    </p>
  </div>,
  <div
    className="imageNotificationWindows"
    id="imageNotificationWindows"
  >
    <img
      alt="logo-sofka"
      src="https://www.sofka.com.co/wp-content/uploads/2020/08/sofka-logo-gradient-white.png"
    />
  </div>,
  <footer
    className="footerNotificationWindows"
    id="footerNotificationWindows"
  />,
]
`;
